ember-cli-build.js

```diff
@@ -7,8 +7,6 @@ module.exports = function(defaults) {
     sassOptions: {
       includePaths: [
         "addon/styles",
-        "node_modules/frontend-basis/sass",
-        "node_modules/frontend-basis/node_modules"

```

Comment:
Yeah i've deleted them since they where causing issues with the sass compiler complaining about missing files in node_modules/frontend-basis/node_modules